docs/reference/esql/functions/types/greatest.asciidoc

```diff
@@ -6,11 +6,16 @@
 |===
 first | rest | result
 boolean | boolean | boolean
+boolean | | boolean
 double | double | double
```

Comment:
Unfortunately, this type table looks a bit broken now.

Why don't we generate also `double | | double`, resp. the same for `ip` and `version`?

`least` has the same problem.

Reply:
As noted [above](https://github.com/elastic/elasticsearch/pull/107865/files#r1587735121), it's due to missing tests (i.e. invoking with just one parameter). The generation, based on the data source, should work as intended, though.